config/filter.d/ignorecommands/apache-fakegooglebot.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Based on: https://isc.sans.edu/forums/diary/When+Google+isnt+Google/15968/
```

Comment:
NB that long ugly cmd name in above is just the first thing which came into my mind ;)